test/e2e/networkpolicy_test.go

```diff
@@ -31,24 +31,31 @@ import (
 )
 
 func TestDifferentNamedPorts(t *testing.T) {
+	// TODO: add check for IPv6 address after NetworkPolicy is supported in IPv6
```

Comment:
nit: s/is supported in IPv6/supports IPv6

(here and in the other TODOs)

Reply:
Done.